addons/snailmail/models/snailmail_letter.py

```diff
@@ -83,8 +83,11 @@ def _compute_display_name(self):
 
     @api.depends('model', 'res_id')
     def _compute_reference(self):
-        for res in self:
-            res.reference = "%s,%s" % (res.model, res.res_id)
+        for rec in self:
+            reference = f''
+            if rec.model and rec.res_id:
+                reference = f"%s,%s" % (rec.model, rec.res_id)
```

Comment:
https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings

Reply:
@ryv-odoo thanks and updated the pr